README.md

```diff
@@ -131,6 +131,37 @@ git push -u origin <your_branch_name>
 
 ---
 
+### Frequently Asked Questions(FAQs) for GSSoC'20:
+
+#### 1. How to start as a beginner so as to contribute to the project?
+There are many beginner level issues to begin with, which include documentation, development and design. Also, feel free to make issues for features as well as for reporting bugs.Participants can claim an issue by commenting that they are interested in it. We are also open to new ideas and suggestions.
+
+
+#### 2. Is this a hardware/IoT project?
+Yes, it is a hardware/IoT + software project. There would be maximum tasks related to frontend, backend, design and documentation of the website and later, it would be integarted with the real-time data from the sensors.
+
+#### 3.How to communicate with us?
+All communication will happen on the Slack channel for this particular projects. Mentors can also be reached through GitHub. Avoid DMs and unnecessary mentions on slack and GitHub both. Checkout this issue for more: <a href="https://github.com/vinitshahdeo/Water-Monitoring-System/issues/15">https://github.com/vinitshahdeo/Water-Monitoring-System/issues/15</a>
+
+#### 4.What is the Tech Stack for this project?
+<strong>Frontend:</strong> ReactJS<br>
+<strong>Backend:</strong> Django, NodeJS
+
+#### 5.What is the deadline for any task submission?
+Mentors can assign issues to participants and also mention a deadline before which the issue is to be resolved. They can reassign issues in case of any delay, so be active and complete your tasks within the negotiated duration.
+
+---
+
+### Important Resources:
+
+<ul>
+  <li> Django Docs: https://docs.djangoproject.com/en/3.0/ </li>
```

Comment:
@PragatiVerma18 Open URLs look weird sometimes.
Can you please remove it?

`[Django](some url)` - > This will become a link. See [this](https://guides.github.com/features/mastering-markdown/).

Use `-` .md syntax for creating lists. See [this](https://guides.github.com/features/mastering-markdown/).